packages/react-inbox/src/hooks/use-inbox.ts

```diff
@@ -36,33 +38,33 @@ export default () => {
             id: "unread",
             label: "Unread",
             filter: {
-              isRead: false
-            }
+              isRead: false,
+            },
           }, {
             id: "all",
             label: "All Messages",
-            filter: {}
-          }]
-        }
+            filter: {},
+          }],
+        },
       });
     },
 
     setCurrentTab: (newTab: string) => {
       dispatch({
         type: "inbox/SET_CURRENT_TAB",
-        payload: newTab
+        payload: newTab,
```

Comment:
lot of lint changes, did we change some rules recently? if so, can we break this PR into lint changes and code change relevant to marking as unread?

Reply:
Removed.